pkg/cmd/server/kubernetes/master/master_config_test.go

```diff
@@ -210,98 +231,113 @@ func (r sortedGCIgnoredResources) Swap(i, j int) {
-			ConcurrentDaemonSetSyncs:                        2,
-			ConcurrentJobSyncs:                              5,
-			ConcurrentResourceQuotaSyncs:                    5,
-			ConcurrentDeploymentSyncs:                       5,
-			ConcurrentNamespaceSyncs:                        10,
-			ConcurrentSATokenSyncs:                          5,
-			ConcurrentServiceSyncs:                          1,
-			ConcurrentGCSyncs:                               20,
-			ConfigureCloudRoutes:                            true,
-			NodeCIDRMaskSize:                                24,
-			ServiceSyncPeriod:                               metav1.Duration{Duration: 5 * time.Minute},
-			ResourceQuotaSyncPeriod:                         metav1.Duration{Duration: 5 * time.Minute},
-			NamespaceSyncPeriod:                             metav1.Duration{Duration: 5 * time.Minute},
-			PVClaimBinderSyncPeriod:                         metav1.Duration{Duration: 15 * time.Second},
-			HorizontalPodAutoscalerSyncPeriod:               metav1.Duration{Duration: 30 * time.Second},
-			DeploymentControllerSyncPeriod:                  metav1.Duration{Duration: 30 * time.Second},
-			MinResyncPeriod:                                 metav1.Duration{Duration: 12 * time.Hour},
-			RegisterRetryCount:                              10,
-			RouteReconciliationPeriod:                       metav1.Duration{Duration: 10 * time.Second},
-			PodEvictionTimeout:                              metav1.Duration{Duration: 5 * time.Minute},
-			NodeMonitorGracePeriod:                          metav1.Duration{Duration: 40 * time.Second},
-			NodeStartupGracePeriod:                          metav1.Duration{Duration: 60 * time.Second},
-			NodeMonitorPeriod:                               metav1.Duration{Duration: 5 * time.Second},
-			HorizontalPodAutoscalerUpscaleForbiddenWindow:   metav1.Duration{Duration: 3 * time.Minute},
-			HorizontalPodAutoscalerDownscaleForbiddenWindow: metav1.Duration{Duration: 5 * time.Minute},
-			HorizontalPodAutoscalerTolerance:                0.1,
-			HorizontalPodAutoscalerUseRESTClients:           true, // we ignore this for now
-			ClusterName:                                     "kubernetes",
-			TerminatedPodGCThreshold:                        12500,
-			VolumeConfiguration: componentconfig.VolumeConfiguration{
-				EnableDynamicProvisioning:  true,
-				EnableHostPathProvisioning: false,
-				FlexVolumePluginDir:        "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
-				PersistentVolumeRecyclerConfiguration: componentconfig.PersistentVolumeRecyclerConfiguration{
-					MaximumRetry:             3,
-					MinimumTimeoutNFS:        300,
-					IncrementTimeoutNFS:      30,
-					MinimumTimeoutHostPath:   60,
-					IncrementTimeoutHostPath: 30,
+	expectedDefaults := &cmapp.KubeControllerManagerOptions{
+		Generic: cmoptions.GenericControllerManagerOptions{
+			SecureServing: &apiserveroptions.SecureServingOptions{
+				BindAddress: net.ParseIP("0.0.0.0"),
+				BindPort:    0,
```

Comment:
slightly surprised.  @sttts no default here?  Planning to supply 10252 later?

Reply:
In 1.10 we have the secure port still disabled by default.

Reply:
https://github.com/sttts/kubernetes/blob/468655b76aac1302092bd11efc6d6337b78c8483/cmd/controller-manager/app/options/options.go#L84